src/components/Icon/Icon.tsx

```diff
@@ -212,22 +218,20 @@ function Icon({
     contentMarkup = <div className={styles.Placeholder} />;
   } else if (React.isValidElement(source)) {
     contentMarkup = source;
-  } else {
-    const iconSource =
-      typeof source === 'string' && isBundledIcon(source)
-        ? BUNDLED_ICONS[source]
-        : source;
-    contentMarkup = iconSource &&
-      iconSource.viewBox &&
-      iconSource.body && (
-        <svg
-          className={styles.Svg}
-          viewBox={iconSource.viewBox}
-          dangerouslySetInnerHTML={{__html: iconSource.body}}
-          focusable="false"
-          aria-hidden="true"
-        />
-      );
+  } else if (isBundledIcon(source)) {
+    const iconSource = BUNDLED_ICONS[source] as SVGSource;
+    contentMarkup = renderSVG(iconSource);
+  } else if (untrusted && isUntrustedSVG(source)) {
+    contentMarkup = (
+      <img
+        className={styles.Img}
+        src={`data:image/svg+xml;base64,${btoa(source)}`}
```

Comment:
I think if you omit the `base64`, then you can output your source as-is without running it through btoa()

```
`data:image/svg+xml;${source}`
```